indexer/Dockerfile.bazooka.remote

```diff
@@ -22,7 +25,7 @@ COPY tsconfig.json ./
 RUN npm install -g pnpm@6
 
 # Install npm modules using pnpm
-RUN pnpm install --loglevel warn --frozen-lockfile
+RUN pnpm install --loglevel warn --frozen-lockfile --unsafe-perm
```

Comment:
`--unsafe-perm` needed, otherwise npm skips the `postinstall` since it thinks its malicious